snapshots/README.md

```diff
@@ -0,0 +1,42 @@
+# Snapshots
+
+## Block [18010657](https://www.mintscan.io/cosmos/block/18010657)
+
+This block is selected to be pre-tally for [cosmoshub-4 proposal 848](https://www.mintscan.io/cosmos/proposals/848)
+
+Block [18010657](https://www.mintscan.io/cosmos/block/18010657) is the last block where vote are present in the state.
+In order to get all votes, we need to add the votes present in block [18010658](https://www.mintscan.io/cosmos/block/18010658) where the tally happens.
+This is because after the tally is completed, the votes a removed from state.
+
+### Download
+
+An export is stored in S3 accessible here: https://atomone.fra1.digitaloceanspaces.com/cosmoshub4/cosmoshub-4-export-18010657.json
```

Comment:
the file isn't accessible for me, request returns 403:

https://atomone.fra1.digitaloceanspaces.com/cosmoshub4/cosmoshub-4-export-18010657.json

Reply:
There was a typo that has been fixed, use the `main` version plz.